projects/configs/uclh-nasogastric-tube-project-ngt-only-full-dataset.yaml

```diff
@@ -0,0 +1,24 @@
+#  Copyright (c) 2022 University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project:
+    name: "uclh-nasogastric-tube-project-ngt-only-full-dataset"
+    modalities: ["DX", "CR"]
+
+tag_operation_files:
+    ["uclh-nasogastric-tube-project-ngt-only-full-dataset-tag-operations.yaml"]
```

Comment:
is there a benefit to having this ending with `tag-operations`?

Reply:
not really I guess, I always prefer not to have duplicate file names, but I'm happy to change if you think this is better